vendor/bat-native-ledger/src/bat/ledger/internal/database/database_server_publisher_info.cc

```diff
@@ -144,6 +145,12 @@ void DatabaseServerPublisherInfo::OnGetRecord(
   info->updated_at = GetInt64Column(record, 2);
   info->banner = banner.Clone();
 
+  // The `CONNECTED` status is deprecated. If this value appears in the database
```

Comment:
Since we are going to uplift I chose not to implement a migration. We can add a migration when we remove the `CONNECTED` state entirely.